lib/elixir/lib/kernel.ex

```diff
@@ -5462,6 +5462,69 @@ defmodule Kernel do
     raise ArgumentError, "modifier must be one of: s, a, c"
   end
 
+  @doc """
+  Handles the `~PID` sigil.
+
+  This function allows creating arbitrary PIDs, even the ones that don't actually exist in the VM.
+  As such, it should only be used for debugging purposes.
+  """
```

Comment:
Can we add an "Examples" section to this and the other sigil functions?

Reply:
Good call. I was skeptical at first because we're showing the same exact code but I think it's an improvement nonetheless. WDYT?

Another thing we could do is maybe make it less boring with:

```
iex> is_pid(~PID<0.108.0>)
true
```

Reply:
I think it's fine to show the same thing but in Elixir we try to have this "Example" section everywhere so even if it's a bit pointless I think it's still nice to have :)